lib/simplecov/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module SimpleCov
-  VERSION = "0.16.1".freeze
+  VERSION = "0.16.2".freeze
```

Comment:
Bumping versions is usually more of a maintainer choice (with a lot of the other fixes I'll probably go for 0.17.0) so I think it's fine to keep as is before result :)